sdk/cognitiveservices/FormRecognizer/README.md

```diff
@@ -0,0 +1,25 @@
+## NOTE: This package has been deprecated and is no longer supported
+
+This package has endpoints that are no longer active and cannot make proper use of the Form Recognizer service. A newer package is available at [Azure.AI.FormRecognizer][new_package] as of 2020-08-20.
+
+# Microsoft Azure Cognitive Services Form Recognizer client library for .NET
+
+## Getting started
```

Comment:
FYI, if you put the path of the project in [this exclusion list](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/.docsettings.yml), the Readme doesn't have to follow the format. I will say because this is track1 that it applies.

Reply:
That's convenient.